source/install/cloud-changelog.md

```diff
@@ -4,12 +4,57 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2022-03-02](#release-2022-03-02)
 - [Release 2022-02-16](#release-2022-02-16)
 - [Release 2022-02-10](#release-2022-02-10)
 - [Release 2022-02-03](#release-2022-02-03)
 - [Release 2022-01-27](#release-2022-01-27)
 - [Release 2022-01-25](#release-2022-01-25)
-- [Release 2022-01-11](#release-2022-01-11)
+
+## Release 2022-03-02
+
+### Highlights
+
+#### Custom User Groups
+ - Added the APIs and app, store, and authorization methods to create and delete custom groups and to add and remove custom group members.
+
+### Improvements
+
+#### User Interface (UI)
+ - Added a use case onboarding checklist which is displayed to first admin users. Also added a status that displays whether the first admin onboarding has been completed and added an in-app migration to determine whether existing installations completed the first admin onboarding.
```

Comment:
Propose we just simplify to:

New onboarding experience for first System Admin user